jpos/src/main/java/org/jpos/util/TPS.java

```diff
@@ -80,7 +80,7 @@ public TPS(boolean autoupdate) {
     public TPS(final long period, boolean autoupdate) {
         super();
         count = new AtomicInteger(0);
-        start = peakWhen = Instant.now();
+        start = peakWhen = Instant.now(NanoClock.systemUTC());
```

Comment:
We don't need NanoClock for monotonic during the living of the same instance of the jvm, `System.nanoTime()` already provides that

Reply:
peakWhen just give us an approximate time where the TPS peaked, it's intended to be used by a human to get an idea of the time of day, and eventually correlate it to other graphs. No need for nano precision.

Reply:
Well I'm trying to ensure the TPS computations are fully monotonic here, hence the use of `NanoClock`.